src/oop/user_defined_classes/hard/q4/QuestionText.md

```diff
@@ -0,0 +1,41 @@
+# oop-user_defined_classes-hard-q4
+
+## Notes
+
+- Vary both of the attributes for this question and the class name
+- Also vary the method name
+- Java question
+
+## Question Text
+
+Have you ever lived in an apartment that has no laundry machines, so you need to walk your dirty clothes to a laundromat
+down the street? Write a Java class called `Laundromat` to keep track of dirty items and let you know when they are 
+clean. See the description below for the requirements.
+
+### Laundromat Class:
+
+Your `Laundromat` class should have the following items:
+
+- Attributes:
+    - An `int` called **gymClothes** which should not be accessible from another class
+
```

Comment:
Mention the visibility modifier for the constructors and methods as well

Reply:
Wouldn't a private constructor for an object be redundant? If no other class can access it? Also, I was using the "which another class can use" in the method description to signify that the method should be accessible from another class

Reply:
They might now add a modifier for constructor which would be the default modifier and is accessible from other classes in the same package. I believe for these questions we would want them to make their constructors public.

Reply:
*They might not add